### PR TITLE
west.yml using zephyr with twister fix

### PR DESCRIPTION
**Pull request by DatGizmo**
Signed-off-by: Sebastian Wezel <sebastian.wezel@nordicsemi.no>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -35,6 +35,8 @@ manifest:
       url-base: https://github.com/alexa
     - name: nordicsemi
       url-base: https://github.com/NordicSemiconductor
+    - name: gizmo
+      url-base: https://github.com/datgizmo
```
Comment (mbolivar-nordic) [inline west.yml:39]:
```suggestion
```

File: west.yml [hunk 2]
```diff
@@ -51,8 +53,9 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/introduction/index.html
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
+      remote: gizmo
```
Comment (mbolivar-nordic) [inline west.yml:56]:
```suggestion
```

File: west.yml [hunk 3]
```diff
@@ -51,8 +53,9 @@ manifest:
...
       repo-path: sdk-zephyr
-      revision: v2.4.99-ncs1
+      revision: 852a2990a8dd70f37b4f0b7dcc80b6b6ff00ab25
```
Comment (mbolivar-nordic) [inline west.yml:58]:
```suggestion
      revision: pull/472/head
```

**Review by mbolivar-nordic — CHANGES_REQUESTED** [review]
@DatGizmo we need to update the manifest in sdk-nrf when your sdk-zephyr PR is merged, and we should be testing against the PR reference, not a SHA in your fork.

Once the sdk-zephyr PR is merged, you will need to update this PR with the final SHA that gets merged into sdk-zephyr master. Then we will merge this sdk-nrf PR and users will have access to the updated sdk-zephyr using `west update`.

**Comment by PerMac** [comment]
I suggested to put [DNM] on it, point in manifest to PR in sdk-zephyr, and when that PR is merged, point to the proper SHA and remove DNM from here. With the rest I agree with you @mbolivar-nordic

**Comment by mbolivar-nordic** [comment]
@DatGizmo thanks for the update! Can you please force push an update to this PR changing the revision to c14c3658222fa940633f4560c9f9e96f92e3824a for sdk-zephyr? I've merged the PR as that commit.

**Comment by DatGizmo** [comment]
The west.yaml was updated to commit "5acae16d9ea808785279b1de1cd177108c1b1354"
Which already includes the "c14c3658222fa940633f4560c9f9e96f92e3824a" commit.
So merging this PR is no longer needed.

**Pull request closed without merging**